### PR TITLE
Add ligand spec

### PR DESCRIPTION
**Pull request by jthorton**
This will add the ANI2x computations to the fragmented ligand benchmark set. We expect the CI to fail here on basis check as some molecules contain I and Br which are not part of the ANI2x model.

I also need to update the conda version of QCSubmit to allow users to submit jobs which raise basis set errors as I am not sure I want to allow this or if we should make a new dataset which will only contain the correct elements.

- [x] QCSubmit validation passed
- [ ] Ready to submit!

### CODE REVIEW

**Comment by jthorton** [comment]
@dotsdl QCSubmit is now updated and I have updated the lifecycle script to use the new `ignore_errors` flag we should be good to start these now when the lifecycle update to use the `compute.json` files is finished.

File: submissions/2020-07-27-OpenFF-Benchmark-Ligands/compute.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      "method": "ani2x",
+      "basis": null,
+      "program": "torchani",
+      "spec_name": "ani2x",
+      "spec_description": "ANI2x ML potential.",
+      "store_wavefunction": "none"
+    }
+  },
+  "dataset_name": "OpenFF-benchmark-ligand-fragments-v1.0",
+  "dataset_tagline": "OpenForcefield TorsionDrives.",
```
Comment (dotsdl) [inline submissions/2020-07-27-OpenFF-Benchmark-Ligands/compute.json:13]: Does the tagline need to match that of the original dataset? Do we want it to?

File: submissions/2020-07-27-OpenFF-Benchmark-Ligands/compute.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  "maxiter": 200,
+  "driver": "gradient",
+  "scf_properties": [
+    "dipole",
+    "quadrupole",
+    "wiberg_lowdin_indices",
+    "mayer_indices"
+  ],
+  "priority": "normal",
+  "description": "A TorsionDrive dataset using geometric.",
```
Comment (dotsdl) [inline submissions/2020-07-27-OpenFF-Benchmark-Ligands/compute.json:24]: Same as above question. I anticipate this doesn't matter, but thought it worth asking. I don't consider it a blocker for submission today.

**Comment by dotsdl** [comment]
Since this is supplemental compute, we care less about checks passing; we know this is exploratory. Thanks @jthorton!

**Comment by dotsdl** [comment]
Moved to "Requires Scientific Review" as this submission is no longer advancing. It may be possible to clear the ~200 `OptimizationRecord` errors due to lack of convergence by increasing the `maxiter` key for the `optimization_procedure` used.

Do you have thoughts on whether this would be effective @JoshHorton? @trevorgokey?

**Pull request merged**